v2.0/check.md

```diff
@@ -4,32 +4,33 @@ summary: The Check constraint specifies that values for the column in INSERT or
 toc: false
 ---
 
-The Check [constraint](constraints.html) specifies that values for the column in [`INSERT`](insert.html) or [`UPDATE`](update.html) statements must return `TRUE` or `NULL` for a Boolean expression. If any values return `FALSE`, the entire statement is rejected.
+The `CHECK` [constraint](constraints.html) specifies that values for the column in [`INSERT`](insert.html) or [`UPDATE`](update.html) statements must return `TRUE` or `NULL` for a Boolean expression. If any values return `FALSE`, the entire statement is rejected.
 
 <div id="toc"></div>
 
 ## Details
 
-- If you add a Check constraint to an existing table, existing values are not checked. However, any updates to those values will be.
-  {{site.data.alerts.callout_info}}In the future we plan to expand the Check constraint to include a check on any existing values in the column.{{site.data.alerts.end}}
-- Check constraints may be specified at the column or table level and can reference other columns within the table. Internally, all column-level Check constraints are converted to table-level constraints so they can be handled consistently.
-- You can have multiple Check constraints on a single column but ideally, for performance optimization, these should be combined using the logical operators. For example
+- If you add a `CHECK` constraint to an existing table, existing values are not checked. However, any updates to those values will be.
+  {{site.data.alerts.callout_info}}In the future we plan to [expand the `CHECK` constraint](https://github.com/cockroachdb/cockroach/issues/23663) to include a check on any existing values in the column.{{site.data.alerts.end}}
```

Comment:
@dt just pointed out this point (and the callout) are plainly wrong. This limitation was lifted in cockroachdb 1.1. 
(in https://github.com/cockroachdb/cockroach/pull/9152 )
One needs to use VALIDATE to check the previous rows.

Reply:
Ok, thanks for clarifying. I updated this bullet point for 2.0 and 1.1